contracts/deposit-service/DepositServiceBase.sol

```diff
@@ -2,13 +2,15 @@
 
 pragma solidity 0.8.9;
 
+import { SafeTokenTransfer } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/SafeTransfer.sol';
 import { IAxelarGateway } from '../interfaces/IAxelarGateway.sol';
 import { IDepositServiceBase } from '../interfaces/IDepositServiceBase.sol';
-import { IERC20 } from '../interfaces/IERC20.sol';
 import { Bytes32ToString, StringToBytes32 } from '../util/BytesStringUtil.sol';
 
 // This should be owned by the microservice that is paying for gas.
-contract DepositServiceBase is IDepositServiceBase {
+abstract contract DepositServiceBase is IDepositServiceBase {
+    using SafeTokenTransfer for address;
```

Comment:
```suggestion
    using SafeTokenTransfer for IERC20;
```
typo? also doesn't look like it's being used here

Reply:
Yup removed